tests/performance/mixed_tensors/data_gen.cpp

```diff
@@ -14,19 +15,23 @@ class RandomStrings {
 private:
     StringVector _strings;
 
-    StringVector gen_strings(size_t count, size_t offset) {
+    StringVector gen_strings(size_t count, size_t offset, bool number_string) {
         StringVector result;
         for (size_t i = offset; i < (count + offset); ++i) {
             char str[7];
-            snprintf(str, 7, "%06d", i);
+            if (number_string) {
+                snprintf(str, 7, "%06d", i);
```

Comment:
I also realised that due to the leading zeroes we are not hitting the optimisation in https://github.com/vespa-engine/vespa/pull/16033. I will change that in a later PR. @arnej27959 @havardpe FYI